sdk/src/main/java/com/hedera/hashgraph/sdk/CustomFeeUpdateTransaction.java

```diff
@@ -0,0 +1,125 @@
+package com.hedera.hashgraph.sdk;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.StringValue;
+import com.hedera.hashgraph.sdk.proto.*;
+import com.hedera.hashgraph.sdk.proto.TransactionResponse;
+import io.grpc.MethodDescriptor;
+import org.threeten.bp.Duration;
+import org.threeten.bp.Instant;
+
+import java.util.LinkedHashMap;
+import javax.annotation.Nullable;
+import java.util.Objects;
+import java.util.List;
+import java.util.ArrayList;
+
+public class CustomFeeUpdateTransaction extends Transaction<CustomFeeUpdateTransaction> {
```

Comment:
Lets name this `TokenFeeScheduleUpdateTransaction` for consistency. If we decide to change this later, we can deprecate this class and make a new one.

Reply:
Ryan was the one who initially suggested I follow the `CustomKey` naming convention.  We need to settle on our naming convention.  I think we should have that conversation tomorrow with Ryan.  Last I heard from Ryan, he was having second thoughts.  I favor the `CustomKey` naming convention, personally, and in that case I think we should use it consistently.  But we'll see how that conversation goes!